tests/Measurement.test.ts

```diff
@@ -0,0 +1,150 @@
+import Measurement from '../src/Measurement'
+import Vector from '../src/Vector'
+import Dimension from '../src/Dimension'
+import { Cx } from '../src/Complex'
+import './customMatchers'
+
+describe('Measurement', () => {
+  it('the simplest measurement', () => {
+    const vector = Vector.fromSparseCoordNames(
+      [
+        ['0', Cx(1)],
+        ['1', Cx(1)],
+      ],
+      [Dimension.qubit()],
+    ).normalize()
+    const m = new Measurement([{ name: [], vector }])
```

Comment:
Just a stylistic change, but because you already have it, you could use `Measurement.fromVector(vector)` here and througout the tests.